cfg.py

```diff
@@ -77,10 +74,7 @@
 def gen_connection_string():
     # if not on Google then use local MySQL
     if IS_PROD or IS_QA:
-        conn_template = (
-            "mysql+mysqldb://%s:%s@localhost:3306/_DB_NAME_?unix_socket=/cloudsql/%s"
-        )
-        return conn_template % (MYSQL_USER, MYSQL_PASS, MYSQL_CONNECTION_NAME)
+        return f"mysql+mysqldb://{MYSQL_USER}:{MYSQL_PASS}@localhost:3306/_DB_NAME_?unix_socket=/cloudsql/MYSQL_CONNECTION_NAME"
```

Comment:
I think this should be: ` return f"mysql+mysqldb://{MYSQL_USER}:{MYSQL_PASS}@localhost:3306/_DB_NAME_?unix_socket=/cloudsql/{MYSQL_CONNECTION_NAME}"`
`MYSQL_CONNECTION_NAME` is treated as a string otherwise.